src/main/pages/che-7/end-user-guide/proc_using-a-badge-to-link-to-workspaces.adoc

```diff
@@ -0,0 +1,27 @@
+[id="using-a-badge-to-link-to-workspaces_{context}"]
```

Comment:
Why does this file have no context defined? I'm confused.

Reply:
It doesn't need it (only files that include other files need context, i.e. assemblies).